src/Npgsql/KerberosUsernameProvider.cs

```diff
@@ -79,7 +79,7 @@ static void DetectUsername()
             _principalWithoutRealm = components[0];
         }
 
-        static string FindInPath(string name) => Environment.GetEnvironmentVariable("PATH")
+        static string FindInPath(string name) => Environment.GetEnvironmentVariable("PATH")!
```

Comment:
Maybe handle the situation where PATH isn't defined? It can theoretically happen :) We can just return null here I think...